lib/modules/platform/github/index.ts

```diff
@@ -942,65 +939,52 @@ export async function getBranchPr(branchName: string): Promise<GhPr | null> {
     branchName,
     state: 'open',
   });
+
   if (openPr) {
     return openPr;
   }
 
-  const autoclosedPr = await findPr({
-    branchName,
-    state: 'closed',
-  });
-  if (
-    autoclosedPr?.title?.endsWith(' - autoclosed') &&
-    autoclosedPr?.closedAt
-  ) {
-    const closedMillisAgo = DateTime.fromISO(autoclosedPr.closedAt)
-      .diffNow()
-      .negate()
-      .toMillis();
-    if (closedMillisAgo > REOPEN_THRESHOLD_MILLIS) {
-      return null;
-    }
-    logger.debug({ autoclosedPr }, 'Found autoclosed PR for branch');
-    if (GlobalConfig.get('dryRun')) {
-      logger.info('DRY-RUN: Would try to reopen autoclosed PR');
-      return null;
-    }
-    const { sha, number } = autoclosedPr;
-    try {
-      await ensureBranchSha(branchName, sha!);
-      logger.debug(`Recreated autoclosed branch ${branchName} with sha ${sha}`);
-    } catch (err) {
-      logger.debug(
-        { err, branchName, sha, autoclosedPr },
-        'Could not recreate autoclosed branch - skipping reopen',
-      );
-      return null;
-    }
-    try {
-      const title = autoclosedPr.title.replace(regEx(/ - autoclosed$/), '');
-      const { body: ghPr } = await githubApi.patchJson<GhRestPr>(
-        `repos/${config.repository}/pulls/${number}`,
-        {
-          body: {
-            state: 'open',
-            title,
-          },
+  return null;
+}
+
+export async function tryReuseAutoclosedPr(
+  autoclosedPr: Pr,
+  branchName: string,
```

Comment:
Wouldn't this be `autoclosedPr.sourceBranch` ?

Reply:
Well, yeah, I think so